packages/orbit-components/src/primitives/MobileDialogPrimitive/MobileDialogPrimitive.stories.tsx

```diff
@@ -1,49 +1,43 @@
 import * as React from "react";
+import { action } from "@storybook/addon-actions";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import Button from "../../Button";
 
-import TooltipPrimitive from ".";
+import MobileDialogPrimitive from ".";
 
-export default {
+const meta: Meta<typeof MobileDialogPrimitive> = {
   title: "MobileDialogPrimitive",
-};
-
-export const Playground = ({
-  content,
-  dataTest,
-  tabIndex,
-  enabled,
-  stopPropagation,
-  removeUnderlinedText,
-  labelClose,
-}) => {
-  return (
-    <TooltipPrimitive
-      content={content}
-      dataTest={dataTest}
-      tabIndex={tabIndex}
-      labelClose={labelClose}
-      enabled={enabled}
-      stopPropagation={stopPropagation}
-      removeUnderlinedText={removeUnderlinedText}
-    >
-      <Button>Open Mobile Dialog</Button>
-    </TooltipPrimitive>
-  );
-};
+  component: MobileDialogPrimitive,
 
-Playground.story = {
   parameters: {
-    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    info: "You can try all possible configurations of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      exclude: ["onShow"],
+    },
+  },
+
+  args: {
+    labelClose: "Close",
+    enabled: true,
+    onShow: action("onShow"),
+    renderInPortal: true,
+    tabIndex: 0,
+    content: "Write your text here.",
+    stopPropagation: true,
+    removeUnderlinedText: false,
+    block: false,
+    lockScrolling: false,
   },
 };
 
-Playground.args = {
-  content: "Write your text here.",
-  dataTest: "test",
-  tabIndex: 0,
-  enabled: true,
-  stopPropagation: true,
-  removeUnderlinedText: false,
-  labelClose: "Close",
+export default meta;
+type Story = StoryObj<typeof MobileDialogPrimitive>;
+
+export const Playground: Story = {
+  render: args => (
+    <MobileDialogPrimitive {...args}>
+      <Button>Open Mobile Dialog</Button>
```

Comment:
I'm thinking if using `Text` component instead of the `Button` wouldn't be more suitable to demonstrate how `removeUnderlinedText` arg works (?). With this current setup, this arg in control panel does nothing.